lib/Dialect/D2M/Transforms/InsertDstRegisterAccess.cpp

```diff
@@ -317,6 +317,10 @@ struct D2MInsertDstRegisterAccessRewriter final
                memref.getDefiningOp())) {
       memref = subView.getSource();
     }
+    if (auto definingOp = memref.getDefiningOp();
```

Comment:
:warning: **llvm\-qualified\-auto** :warning:
`` auto definingOp `` can be declared as `` auto *definingOp ``
```suggestion
    if (auto *definingOp = memref.getDefiningOp();
```